### DISCUSSION TITLE
Infinite/collection scroll with multiple pubs on the same page (blog style)

### DISCUSSION THREAD

**Discussion by maxlecalve** [knowledgefutures/pubpub#1894] [Ideas]
Hi there,
 I'm looking for a possibility to view multiple pubs on one single scroll-able page. 
Here is the case I have in mind: I want to transfer an online exhibition created on WordPress to a more perennial and citable medium, with images, video and text, like a scrollable catalog with multiple entries. 
Here is the page that I would like to convert (with added text from the "artwork" section):
https://antart.easaonline.org/home/

in the end, I guess it would look very much like a blog feed. Is there a way to get to this result?
I think that not being able to scroll through short pubs is a drawback inherited from the academic article model. I can imagine that it would be a lot of work, but having multiple options to deal with the headers between the individual pubs would be ideal. 
I'm a big fan of PubPub, thanks for the amazing work!

**Comment by gabestein** [comment]
Hello — you could certainly use a Page (or Collection Layout) with some custom html and styling to come close to that layout. We have some similar examples, e.g. Contours (https://contours.pubpub.org/clinard). But it would require some custom HTML work and knowledge of CSS. It also wouldn't use Pubs, so you'd lose some of the citability and depositing features.

In the medium term, we are thinking about ways to generalize content types on PubPub to allow you to get the academic and metadata features of a Pub with more flexible layout options. But at the moment, it's not the way our architecture works, and not in our near-term plans to support.

**Reply by maxlecalve** [reply]
Many thanks for the reply, we will speak with our design team.
Just one more thought: I have several other pubpub projects that are designed to be collections of short texts, which should remain citable individually. These would all greatly benefit from the possibility of navigation between them through scrolling. 
Scrolling down to the next pubpub would be neat. Thanks again for the great work!
